test/integration/utils_test.go

```diff
@@ -179,3 +220,15 @@ func setJobCompletedStatus(job *batchv1.Job) {
 		Status: corev1.ConditionTrue,
 	})
 }
+
+func logPods(t *testing.T) {
+	podList := &corev1.PodList{}
+	require.NoError(t, testK8sClient.List(testCtx, podList, client.InNamespace(testNS)))
+	fmt.Println("=== Pods ===")
```

Comment:
Alternatively, you can use `t.Log` here and below so that this is only printed on failures

Reply:
keeping as fmt.Print b/c its hard to read logged yaml